src/library_browser.js

```diff
@@ -1381,6 +1381,35 @@ var LibraryBrowser = {
     exit(status);
   },
 
+  emscripten_get_window_title__proxy: 'sync',
+  emscripten_get_window_title__sig: 'iv',
```

Comment:
this should be `vi` I think, not `iv`?

Reply:
Doesn't the function signature indicate that it's currently correct?
`char *emscripten_get_window_title();`

However, this comment made me realize that the function signature in the documentation is incorrect, so I'll fix that.

Reply:
Oh, yes, maybe that's what confused me.